packages/contracts/src/interfaces/IRelayRegistrar.sol

```diff
@@ -56,20 +62,27 @@ interface IRelayRegistrar is IERC165 {
      * @return `true` if this `RelayRegistrar` keeps registrations on-chain in storage in addition to emitting events.
      * `false` if only events are emitted as part of Relay Server registration on this `RelayRegistrar`.
      */
-    function isUsingStorageRegistry() external returns (bool);
+    function getIsUsingStorageRegistry() external returns (bool);
```

Comment:
no need to rename here: the standard spec for "getter" is a method that starts with "get" or "is"